wandb_workspaces/reports/v2/interface.py

```diff
@@ -235,32 +319,54 @@ class Link(Base):
 
 @dataclass(config=dataclass_config, repr=False)
 class InlineLatex(Base):
+    """A block of inline LaTeX. Does not add newline
```

Comment:
Avoid the term `block` here which is confusing because it has a very specific meaning in Reports.

This is just inline latex